tokenizers/src/utils.rs

```diff
@@ -50,6 +50,16 @@ pub enum TruncationStrategy {
     OnlySecond,
 }
 
+impl std::convert::AsRef<str> for TruncationStrategy {
```

Comment:
Also I added that to get a string representation of the enum (same for `PaddingDirection`), my first ever trait implementation, obviously feel free to let me know if there's a better way!